cmd/neofs-ir/main.go

```diff
@@ -57,8 +59,33 @@ func main() {
 	ctx := grace.NewGracefulContext(log)
 	intErr := make(chan error) // internal inner ring errors
 
-	pprof := profiler.NewProfiler(log, cfg)
-	prometheus := profiler.NewMetrics(log, cfg)
+	var httpServers []*httputil.Server
+
+	for _, item := range []struct {
```

Comment:
Encapsulate filling `httpServers` in separate function down below. This will make main function a bit smaller and cleaner. Something like
```go
func initHTTPservers(cfg viper.Viper) []*http.util.Server {
 . . .
}
```

Reply:
Done.

We can do the same for serve/shutdown, although their code is much smaller.